.github/workflows/docker_build_tpls.yml

```diff
@@ -1,30 +1,131 @@
-# This is a basic workflow that is manually triggered
-
 name: Docker TPL build
 
-# Controls when the action will run. Workflow runs when manually triggered using the UI
-# or API.
+# Manually triggered
 on:
   workflow_dispatch:
-    # Inputs the workflow accepts.
-    inputs:
-      name:
-        # Friendly description to be shown in the UI instead of 'name'
-        description: 'Person to greet'
-        # Default value if no value is explicitly provided
-        default: 'World'
-        # Input has to be provided for the workflow to run
-        required: true
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+    # No input required
+
 jobs:
-  # This workflow contains a single job called "greet"
-  greet:
-    # The type of runner that the job will run on
+  clang-10:
     runs-on: ubuntu-latest
+    name: Builds a Clang 10 docker image and extracts generated hostconfigs
+    env:
+      TAG: clang-10
+      REPO: axom/tpls
+      HOSTCONFIG_LOC: /home/axom/export_hostconfig
+      DOCKERFILE_PREFIX: ./scripts/docker/dockerfile_
+    steps:
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    - name: Get dockerfile name
+      shell: bash
+      run: echo "##[set-output name=filename;]$(echo ${DOCKERFILE_PREFIX}${TAG})"
+      id: dockerfile_name
+    - name: Get dockerhub repo name
+      shell: bash
+      run: |
+        echo ${REPO}:${TAG}_`date "+%m-%d-%y_%Hh-%Mm"`
```

Comment:
Note: the time stamp appears to be UTC:
I.e. a job from a few hours ago ( 12/1/2020 ~ 4 PM PT) labels its TPL as:
```
Status: Downloaded newer image for axom/tpls:clang-10_12-02-20_00h-05m
```

Reply:
We are using it mainly as a unique identifier.  Do you want it more accurate in the PST sense?

Reply:
No, I think UTC is probably the right choice.  I was just pointing it out.